Library/Homebrew/cmd/list.rb

```diff
@@ -116,8 +116,14 @@ def list
       ls_args << "-r" if args.r?
       ls_args << "-t" if args.t?
 
-      safe_system "ls", *ls_args, HOMEBREW_CELLAR unless args.cask?
-      list_casks(args: args) unless args.formula?
+      unless args.cask?
+        ohai "Formulae:" if $stdout.tty?
+        safe_system "ls", *ls_args, HOMEBREW_CELLAR
+      end
+      unless args.formula?
```

Comment:
```suggestion
      end
      puts if $stdout.tty? && !args.cask? && !args.formula?
```

Reply:
Are you looking for a space between the formulae and casks? This suggestion is giving me a syntax error 😅

Reply:
Yes, that's the goal!